homeassistant/helpers/config_validation.py

```diff
@@ -402,3 +402,9 @@ def validator(value):
     [vol.Any(SERVICE_SCHEMA, _SCRIPT_DELAY_SCHEMA, EVENT_SCHEMA,
              CONDITION_SCHEMA)],
 )
+
+
+def string_schema(*args: Sequence) -> Dict:
+    """Return a schema dict of Required values or Optional values."""
+    return {val if isinstance(val, vol.Optional) else vol.Required(val): string
+            for val in args}
```

Comment:
15-20% should be enough to include this then?

It also handles Optional strings, but in Voluptuous syntax
```python
CC = 'c'
str_schema('a', 'b', vol.Optional(CC, default='1'))
```

To cater for default shorthand we could add this syntax:
```python
CC = 'c'
str_schema('a', 'b', defaults={CC: '1'})
```